### ISSUE TITLE
Plans page(s): Remove info icons

### ISSUE THREAD

**Issue by boonerang** [Automattic/wp-calypso#37531]
In research sessions customers with visual and cognitive disabilities have struggled to identify and activate the tool-tip. When found by customers, they questioned the value the tool-tip provided. (The content of the tool-tips varies from aspirational comments on the feature to purely duplicative rehashing of the feature.) If there is copy vital to the feature or plan, it should be in the table and not hidden behind an icon that is hard to see and/or activate.

![Screenshot 2019-11-12 14 42 24](https://user-images.githubusercontent.com/6179338/68705753-a618e180-055c-11ea-97a8-aec322c9b66b.png)

![Screenshot 2019-11-12 14 56 11](https://user-images.githubusercontent.com/6179338/68705763-aa44ff00-055c-11ea-8b5c-9d61c724b9c8.png)

Multiple tool-tips could already be removed as they offer no unique new information. For example:

![Screenshot 2019-11-12 14 53 43](https://user-images.githubusercontent.com/6179338/68705806-b92bb180-055c-11ea-803b-53fb1ef02470.png)

![Screenshot 2019-11-12 14 45 03](https://user-images.githubusercontent.com/6179338/68705862-d19bcc00-055c-11ea-99ca-36d03e67de75.png)

Ideally, each feature should have a description clear enough no further details are needed. 

For pages like https://wordpress.com/start/plans having any unique key features added in the text of the column itself gives it a better chance of being read.

For pages like https://wordpress.com/pricing/?ref=logged-out-homepage-lp they are already incredibly long and dense. The usability of multiple tool-tips decreases rapidly as they become “noise” on the page and/or look like part of the design.

#### Steps to reproduce
Visit

1. https://wordpress.com/start/plans 
2. https://wordpress.com/pricing/?ref=logged-out-homepage-lp 

Mouse over any of the 'info' icons.

#### Context / Source

#userresearch 

See also: https://github.com/Automattic/wp-calypso/issues/34604 for some attention being paid to these tables by @lcollette and others.

#### Suggested Fix

Remove the information icons from the tables

**Comment by worldomonation** [comment]
Hi! I'm performing triage on older bugs as part of Quality Squad's efforts to cut down on backlog issue counts.

As there has been no recent activity on this issue it is being closed with the label `Closed - Won't Fix`.
If this issue should remain open, please feel free to re-open this bug.
